app/views/profiles/_edit.haml

```diff
@@ -9,10 +9,22 @@
   = render 'profiles/edit_public', :profile => profile, :aspect => aspect, :person => person
 
   %hr
+  :javascript
```

Comment:
It matches "= check_box_tag 'profile[is_public]', true, profile.is_public", line 64 of paste.

It doesn't really matter, because switchAdditionalVisibility function doesn't get called at all.